packages/cli/README.md

```diff
@@ -0,0 +1,13 @@
+# NGXS CLI
+
+## Started
+```shell
+$ npm install @ngxs/cli -g
+```
+
+```shell
+$ ngxs
+? Write store name, please... 
+```
+
+![](https://habrastorage.org/webt/g5/td/dz/g5tddzflkkwftfvpvim3ckwveau.gif)
```

Comment:
Should rather move the gif locally and use that one as opposed to the external link.

Reply:
Agreed. Just add it to the repo.

Reply:
done